torch/csrc/distributed/c10d/Work.hpp

```diff
@@ -144,15 +144,18 @@ class TORCH_API Work : public torch::CustomClassHolder {
 struct TORCH_API WorkInfo {
   WorkInfo(
       const OpType& opType,
+      const uint64_t& seq,
```

Comment:
it's better to pass primitive type by value?

Reply:
> it's better to pass primitive type by value?

yep, thanks!